### PR TITLE
Fixed indentation issue for python code

### PR DESCRIPTION
**Pull request by g-paras**
In order to avoid excessive Indentation #1 , program will type code line by line instead of typing whole code at once. This  can be done by spliting the code by new line character `\n`, type that line then press `Enter` key for next line and then press `Home` to move the cursor at the start of the line and repeat the same process.

It is a bit slower but working fine.

### CODE REVIEW

**Comment by tushar5526** [comment]
Nice workaround!
You can also remove the NOTE I have added in Readme.md, that there are issues in indentation.

File: simulate_keyboard.py [hunk 1]
```diff
@@ -12,16 +12,21 @@ def type(path=None, delay=None):
...
-            here or provide a path to the file 
+class Complex:
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
     """
     if path:
         with open(path, 'r') as file:
             code = file.read()
             print(code)
```
Comment (tushar5526) [inline simulate_keyboard.py:23]: While you are at it, remove the line `print(code)` too.

**Comment by g-paras** [comment]
- [x] Remove print statement
- [x] Remove NOTE in the README.md

**Comment by tushar5526** [comment]
Merging.
Thanks @g-paras

**Comment by tushar5526** [comment]
Tip: If you mention Closes #2 in the description of your PR. It will automatically close that issue also.

**Pull request merged**